### ISSUE TITLE
Internationalization bundles 

### ISSUE THREAD

**Issue by adrianovieira** [google/docsy#263]
It appears that there are some text which doesn't allow translation.

One of them is feedback page, which have some hard coded texts, e.g.:

```
<p class="feedback--question">Was this page helpful?</p>
<button class="feedback--answer feedback--answer-yes">Yes</button>
<button class="feedback--answer feedback--answer-no">No</button>
```

Do you have any plan to change things like that to also be translated via a bundle inside `/i18n`?

**Comment by LisaFC** [comment]
Yes, we have an existing issue #175 for making the feedback internationalizable, and an open PR with the start of a fix #179 

I think the issue with the existing provisional fix was that it meant you had to enable and disable feedback per language rather than just once for the entire project, I'll take a look and see if I can improve it. Or if anyone else is interested, maybe take a look?

**Comment by chalin** [comment]
Closing this in favor of #175.
